.github/workflows/ci.yaml

```diff
@@ -12,29 +12,16 @@ env:
   CARGO_TERM_COLOR: always
 jobs:
   rustfmt:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: Swatinem/rust-cache@v2
-        with:
-          workspaces: |
-            "containerd-shim-* -> target"
-      - run: 
-          rustup toolchain install nightly --component rustfmt
-      - name: Setup build env
-        run: |
-          make setup
-      - name: fmt
-        run: |
-          make fmt
+    uses: ./.github/workflows/action-check.yml
+  build:
+    uses: ./.github/workflows/action-build.yaml
   build-wasm-images:
-    uses: ./.github/workflows/docker-build-push.yaml
+    uses: ./.github/workflows/action-docker-build-push.yaml
+    needs: build
     with:
       test: true
-  build:
-    uses: ./.github/workflows/build.yaml
   publish-node-installer-image:
-    uses: ./.github/workflows/node-installer.yaml
+    uses: ./.github/workflows/action-node-installer.yaml
     needs: build
     # This action requires use of the GITHUB_TOKEN to publish the image
     # By default, PRs from forks don't have access, so we only run when the PR branch is on origin.
```

Comment:
It looks like this ran on the PR. Is that expected? From the comment it sounds like it shouldn't on forks?

Reply:
oh looks like your branch is on the upstream repo here...